homework-2/essay.md

```diff
@@ -0,0 +1,14 @@
+# The best holidays in my life!
+
+Hey... I'm still Dmitry, and I don't have a classification of my holidays. So today I tell you about my holidays in Spain.
+
+It was randomly. I and my friend wanted to spend the holidays outside of Russia and we looked for a tour with all-inclusive. We found a tour to Spain, to Mallorca. We bought a tour, made a visa and started to prepare.
+
+When we flew there and came to the hotel we found out that we live on the German coast. As it turned out tourist Mallorca divided on the two coasts - a German coast and an English coast. We bought two excursions - "A journey for Mallorca" and "Sea caves".
+
+At the first one, we traveled to Lluc monastery, to pearl factory, and visit a Spanish family that fed us pilaf with mollusks and gave to drink wine. It was fine. 
+At the second one, we traveled to sea caves. At the start, we sat in the motorboat and sailed to the caves. Then we dressed up to the hydro suit, jumped from the motorboat. We swam under the water and surfaced in the cave. Then we went from the entrance to the exit and saw stalactites and stalagmites. After that, we sat in the motorboat and sailed to the port. The organizers of the excursion offered us bought the flash-card with photos and videos from the excursion (5 euro for a card). I bought the flash-card and told to people "Don't buy the flesh card. Just give me some money and e-mail and I will send you photos and videos when I will back home". Some people agreed and gave me some money and e-mails. But the flash-card didn't pay off.
+
+Also, we had a free bar in the hotel. One day my friend drank a lot of alcohol and we went for a walk by the evening city. When my friend was drunk he started to shout "Fuck you, niggers!". I was feared that black people heard him and I shut his mouth. Then I told him that if he will shout - black people will beat him. He calmed down. To a fortune, all ended well.
```

Comment:
- went for an evening walk
- I was afraid
- if he would have shout, people had beaten (conditionals, we're going to learn it someday) OR "if he shouts, people will beat him"
- FORTUNATELY